.github/workflows/turing.yaml

```diff
@@ -180,7 +180,7 @@ jobs:
           cache-dependency-path: ui/yarn.lock
 
       - name: Install
-        run: yarn install --network-concurrency 1
+        run: NODE_OPTIONS=--openssl-legacy-provider yarn install --network-concurrency 1
```

Comment:
Somehow I need an older SSL provider https://github.com/caraml-dev/turing/actions/runs/9969015761/job/27547281483#step:5:35, but I have no idea why since the dependencies in https://github.com/gojekfarm/react-lazylog/blob/master/package.json and in https://github.com/mozilla-frontend-infra/react-lazylog/blob/master/package.json are the same.

I simply followed one of the steps suggested in https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported to fix this.